src/oops/SOLID/singleResponsibilityPrinciple/before/SaveEmployeesMain.java

```diff
@@ -10,7 +11,12 @@ public static void main(String[] args) {
 
         // Save all
         for (Employee e : employees){
-            Employee.save(e);
+            try {
+                repository.save(e);
+            } catch (IOException ioException) {
+                ioException.printStackTrace();
+                System.out.println("Failed to same employee :" + e.getFullName());
```

Comment:
To keep logging at a place, it would be good to move the logs out, using a common class.

Reply:
Agreed, It could be moved out as the complexity increases.